packages/api/cms-api/src/common/filter/mikro-orm.ts

```diff
@@ -95,6 +96,12 @@ export function filterToMikroOrmQuery(
         if (filterProperty.isAnyOf !== undefined) {
             ret.$in = filterProperty.isAnyOf;
         }
+    } else if (filterProperty instanceof ManyToManyFilter) {
+        if (filterProperty.contains !== undefined) {
+            ret.id = {
```

Comment:
hardcoding id here means the primary key of the related entity must be `id`. Far from ideal, but we currently don't have any entity metadata in this function.

Reply:
Should we add an options argument to override the key? Is this even necessary?

Reply:
in practice it's called id in maybe 99%

Reply:
Then we add support to override it if we ever need it.